doc/articles/features/working-with-accessibility.md

```diff
@@ -90,13 +90,10 @@ When the value of `WinRTFeatureConfiguration.Accessibility.HighContrast` is chan
 
 ## Known issues
 
-- `Hyperlink` in `TextBlock` is not supported.
-- `TextBox` and `PasswordBox` don't use `Header` and `PlaceholderText`.
 - `ItemsControl` doesn't use `AutomationProperties.Name` on its `DataTemplate`'s root.
 - There are XAML code generation conflicts between `x:Name` and `AutomationProperties.Name`.
 - A child with the same accessible name as its parent is accessibility focusable.
 - `Control` doesn't receive focus when accessibility focused.
-- `TabIndex` is not supported.
```

Comment:
Are you sure this is supported from a11y point of view?
i.e, if you use TalkBack will it respect TabIndex? I thought TabIndex implementation currently works with focus manager only for tab navigation?

Reply:
Good point, I read these only from the implementation perspective, not A11y perspective. Makes sense to keep it there then